Scripts/Python/nxusBookMachine.py

```diff
@@ -236,6 +238,7 @@
      'GuildPub-Maintainers' : ("Nexus.Messages.GuildPubFull", "Nexus.Messages.GuildPubPopulation"),
      'GuildPub-Messengers' : ("Nexus.Messages.GuildPubFull", "Nexus.Messages.GuildPubPopulation"),
      'GuildPub-Writers' : ("Nexus.Messages.GuildPubFull", "Nexus.Messages.GuildPubPopulation"),
+     'GoMePubNew' : ("Nexus.Messages.GoMePubNewFull", "Nexus.Messages.GoMePubNewPopulation"),
```

Comment:
Does `Nexus.Messages.GoMePubNewFull` exist in the Nexus LOC files already? I don't see a PR for adding this message.

Reply:
Hrm...I think I did add that somewhere. Need to recheck.

Reply:
PR https://github.com/H-uru/moul-assets/pull/130 should properly update the Nexus LOCs for the Pub messages (at least I hope they do after proper review!).